tools/releaser/breaking_changes/v20240530002.md

```diff
@@ -0,0 +1,54 @@
+# Breaking Changes
+
+## SDK changes
+
+go.mongodb.org/atlas-sdk/v20240530002/admin
+Incompatible changes:
+
+- (\*DiskBackupExportJob).GetComponents: changed from func() []DiskBackupBaseRestoreMember to func() []DiskBackupExportMember
+- (_DiskBackupExportJob).GetComponentsOk: changed from func() (_[]DiskBackupBaseRestoreMember, bool) to func() (\*[]DiskBackupExportMember, bool)
+- (\*DiskBackupExportJob).SetComponents: changed from func([]DiskBackupBaseRestoreMember) to func([]DiskBackupExportMember)
+- (*InvoicesApiService).ListInvoicesExecute: changed from func(ListInvoicesApiRequest) (*PaginatedApiInvoice, *net/http.Response, error) to func(ListInvoicesApiRequest) (*PaginatedApiInvoiceMetadata, \*net/http.Response, error)
+- (\*LiveMigrationRequest).GetDropEnabled: removed
+- (\*LiveMigrationRequest).GetDropEnabledOk: removed
+- (\*LiveMigrationRequest).SetDropEnabled: removed
+- AuthorizeCloudProviderAccessRoleApiParams.CloudProviderAccessAWSIAMRoleUpdateRequest: removed
+- CloudProviderAccessAWSIAMRoleRequest: removed
+- CloudProviderAccessAWSIAMRoleUpdateRequest: removed
```

Comment:
Effects of revert when fix for the issue caused multiple fields to be brolen.
Original issue is still present. @AgustinBettati have reported it initially. 

To be established if that is not a blocker for release.